ext/storage_sources/azure_store/README.md

```diff
@@ -0,0 +1,83 @@
+# WiredTiger's Azure Extension
+## 1. Introduction
+This extension allows WiredTiger storage source extensions to read from and write to objects stored
+in Azure Blob Storage using WiredTiger’s provided internal abstraction for storing data in an object
+storage service.
+
+## 2. Building and running
+This section describes how to build WiredTiger with the Azure extension enabled.
+
+### Requirements
+* CMake 3.13 or higher
+* G++ 8.4 or higher
```

Comment:
If clang 6.0.0 is the lowest version you can test with, I'd just list that. The way it's written right now makes it sound like to need GCC, and Clang won't work for some reason.

Reply:
sounds good :) added in https://github.com/wiredtiger/wiredtiger/pull/8802/commits/c2e873fdea9905eb5c0f4c467f5934037aedd4e3

Reply:
Thanks!